Source/Microsoft.Teams.Apps.CompanyCommunicator/Models/SentNotification.cs

```diff
@@ -30,5 +31,20 @@ public class SentNotification : BaseNotification
         /// Gets or sets the number of recipients who were throttled out.
         /// </summary>
         public int Throttled { get; set; }
+
+        /// <summary>
+        /// Gets or sets Teams audience id collection.
```

Comment:
Change "id" in this so "... Teams audience name collection"

Reply:
Addressed.